crates/holochain_sqlite/src/sql/cell/schema/2.sql

```diff
@@ -88,9 +88,12 @@ CREATE TABLE IF NOT EXISTS DhtOp (
     hash             BLOB           PRIMARY KEY ON CONFLICT IGNORE,
     type             TEXT           NOT NULL,
     basis_hash       BLOB           NOT NULL,
-    action_hash      BLOB           NOT NULL,
     require_receipt  INTEGER        NOT NULL,      -- BOOLEAN
 
+    -- This is not strictly an action hash, but a foreign key to a row in the Action table.
+    -- This may be a WarrantHash if the corresponding row in Action is a warrant.
+    action_hash      BLOB           NOT NULL,
```

Comment:
does sqlite care about column ordering? Does this require a migration?

Reply:
The full schema scripts are only for reference, they never get executed. The `n-up.sql` migrations are what actually build up the schema, starting from `0.sql`